src/SumoCoders/FrameworkCoreBundle/Helper/InstallerHelper.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Helper;
+
+use Composer\IO\IOInterface;
+use Symfony\Component\Yaml\Dumper;
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Parser;
+use Symfony\Component\Yaml\Yaml;
+
+class InstallerHelper
```

Comment:
I would fully remove the helper namespace, yes. SumoCoders\FrameworkCoreBundle\Installer\Installer.php seems good to me.